src/CMakeVersion.txt

```diff
@@ -0,0 +1,14 @@
+set(GIT_CMD "git")
+set(GIT_ARGS "log" "-n" "1" "--pretty=format:%h - %ad")
+execute_process(COMMAND ${GIT_CMD} ${GIT_ARGS}
+    RESULT_VARIABLE GIT_RESULT
+    OUTPUT_VARIABLE GIT_COMMIT)
+    if (GIT_RESULT)
+        set (GIT_COMMIT "UNKNOWN")
+    endif()
+set(ZMAP_VERSION "Development Build. Commit ${GIT_COMMIT}")
+
+configure_file(topt.ggo.in topt.ggo @ONLY)
```

Comment:
1) This was not checked against out of source builds (which don't seem to be supported by the project at all)

2) Modifying the .ggo.in files is fine? What is your concern? 

3) Doing this inside the build system seemed preferable to a shell script, especially given that shell scripts aren't used elsewhere in the project.